pkgs/tools/compression/upx/default.nix

```diff
@@ -1,31 +1,14 @@
-{ lib, stdenv, fetchurl, ucl, zlib, perl }:
+{ lib, stdenv, fetchurl, cmake }:
 
 stdenv.mkDerivation rec {
   pname = "upx";
-  version = "3.96";
+  version = "4.0.1";
   src = fetchurl {
     url = "https://github.com/upx/upx/releases/download/v${version}/${pname}-${version}-src.tar.xz";
```

Comment:
Is there any difference between this and the source from github?

Reply:
I updated the PR to use `fetchFromGitHub`.